### ISSUE TITLE
"Exception while updating neighbors" crash

### ISSUE THREAD

**Issue by Dayyer1** [DaedalusGame/EmbersRekindled#145]
Hi, I got this crash when starting up a world. It crashed before it loaded. I also got the message before the crash about how the world may be corrupted and a backup level.dat file is being used. (I don't remember exactly what it says, but I can produce it again if needed.)

The crash happened after I updated from Embers Rekindled versions 1.12 to 1.13. Switching back to 12 creates no problem.
Using forge 2824

[crash-2019-04-02_14.10.26-server.txt](https://github.com/DaedalusGame/EmbersRekindled/files/3035684/crash-2019-04-02_14.10.26-server.txt)

**Comment by DaedalusGame** [comment]
these are all netherquartz crops from mystical agriculture and embers isn't mentioned anywhere in the stacktrace so i don't know how i can help

**Comment by Dayyer1** [comment]
That's strange. Should I report it to them instead?

**Comment by DaedalusGame** [comment]
yeah, unless you accidentally uploaded the wrong crash report that is

**Comment by Dayyer1** [comment]
[crash-2019-04-03_12.03.26-server.txt](https://github.com/DaedalusGame/EmbersRekindled/files/3039937/crash-2019-04-03_12.03.26-server.txt)

What about this one?

**Comment by Dayyer1** [comment]
Nevermind, another MA issue in that one. I'll contact MA.

**Comment by Dayyer1** [comment]
[crash-2019-04-03_12.33.37-server.txt](https://github.com/DaedalusGame/EmbersRekindled/files/3040005/crash-2019-04-03_12.33.37-server.txt)

Ok, I removed MA and got this crash, which mentions Embers.

**Comment by DaedalusGame** [comment]
this honestly looks like an id-shift corruption.

**Comment by Dayyer1** [comment]
Is there a way to fix that?
